modules/custom/activity_basics/src/Plugin/ActivityContext/ContentInMyGroupActivityContext.php

```diff
@@ -120,12 +121,15 @@ public function getRecipients(array $data, int $last_id, int $limit): array {
           return $recipients;
         }
 
-        $node = $group_content->getEntity();
+        $entity = $group_content->getEntity();
```

Comment:
maybe its better to use a different name here since this is not the same `$entity` that is used above.

Reply:
These two entities don't use in parallel in the following code so I don't think that we should spend extra memory on that.